docs/RUNNING_LOCALLY.md

```diff
@@ -75,7 +78,7 @@ DATABASE_URL=psql://gitcoin_user:password@localhost:5432/postgres
 
 ```shell
 
-~/web$ python3 -m virtualenv gcoin
+~/web$ virtualenv -p python3.7 gcoin
```

Comment:
Why would we use virtualenv directly instead of the built in python3 venv commands? just curious

Reply:
Since, we are using f strings in the project, which are supported by python 3.6+ hence can't use default python version hence using this is better solution IMO.

Reply:
Ok, so this is to make sure that python3.7 is used? Makes sense, thanks!